src/Raven.Studio/typescript/models/database/tasks/ongoingTaskModel.ts

```diff
@@ -22,8 +23,19 @@ abstract class ongoingTaskModel {
         
         return (preferredMentor && currentNode) ? preferredMentor !== currentNode : false;
     });
+    
+    abstract get studioTaskType(): StudioTaskType;
+
+    static getStudioTaskType(taskListItem: Raven.Client.Documents.Operations.OngoingTasks.OngoingTask): StudioTaskType {
+        if (taskListItem.TaskType === "QueueEtl") {
+            const task = (taskListItem as Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskQueueEtlListView);
+            return task.BrokerType === "Kafka" ? "KafkaQueueEtl" : "RabbitQueueEtl";
+        }
 
-    static mapTaskType(taskType: Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskType): TasksNamesInUI {
+        return taskListItem.TaskType;
+    }
+    
+    static mapTaskType(taskType: StudioTaskType): TasksNamesInUI {
```

Comment:
let's remove TaskNamesInUi type - and name this method: formatStudioTaskType - it should return string and be only used in view. We don't want to deal with formated values anywhere else.

Reply:
done